dashboard/app/mailers/peer_review_mailer.rb

```diff
@@ -1,4 +1,5 @@
-class PeerReviewMailer < ActionMailer::Base
+class PeerReviewMailer < ApplicationMailer
+  include ActionMailerMetrics
```

Comment:
You don't need this since it's already included in the parent class `ApplicationMailer`